lesson02/assignment/css/_components.content.scss

```diff
@@ -6,12 +6,55 @@ html {
 *:after,
 *:before {
     box-sizing: inherit;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.profile-pic {
+    border-radius: 50%;
+    max-width: 75%;
+}
+
+.card {
+  background-color: $white;
+  max-width: 86%;
```

Comment:
Typically we'd like to give components some sort of hard `max-width` or place it inside a container with one, otherwise we'd end up with much larger than expected components on very wide screens. I'd swap that to a hard value (px, rem, etc) and give it a margin left/right of auto to center.